NuRadioReco/framework/parameter_storage.py

```diff
@@ -0,0 +1,101 @@
+import NuRadioReco.framework.parameters
+
+import itertools
+import logging
+logger = logging.getLogger('NuRadioReco.framework.parameter_storage')
+
+
+class ParameterStorage:
```

Comment:
I'm still curious if it's worth making this module private, because this module should only be used internally, so this somewhat 'clutters' the documentation. I can test this on Monday.

Reply:
I think it is a good idea but maybe for another PR?

Reply:
I have already merge this branch into other branches so I would really like doing this in another PR.

Reply:
I'm not a fan of merging public API and then making it private after, but I guess we can live with a lower standard of backwards-compatibility on `develop`.